CHANGES.rst

```diff
@@ -78,6 +78,8 @@ Bug Fixes
 - Fixed loading data via the Import Data button on top-left of the application.
   [#1608]
 
+- Floating dropdowns are now attached to their menus [#1673]
```

Comment:
I think @rosteen had a beef between drop-down vs dropdown but I forgot which one. 😆 

```suggestion
- Floating dropdowns are now attached to their menus. [#1673]
```

Reply:
Oh really? 👀

Reply:
I think the conclusion when I was trying to figure out which one we should consistently use was that we shouldn't use either! At least that was the style guidance I found online. Not a big deal in the changelog rather than the docs, but perhaps something like this in general is better "Floating menus are now attached to their selector element".

Reply:
Happy to; changed it to the verbiage you described @rosteen